packages/forklift-console-plugin/src/modules/Providers/modals/EditProviderURL/OpenshiftEditURLModal.tsx

```diff
@@ -19,8 +19,8 @@ export const OpenshiftEditURLModal: React.FC<EditProviderURLModalProps> = (props
       <div className="forklift-edit-modal-field-error-validation">
         <Trans t={t} ns="plugin__forklift-console-plugin">
           Error: The format of the provided URL is invalid. Ensure the URL includes a scheme, a
-          domain name, and, optionally, a port. For example: {'<strong>'}
-          https://api.&#8249;your-openshift-domain&#8250;:6443{'</strong>'}.
+          domain name, and, optionally, a port. For example:{' '}
+          <strong>https://api.&#8249;your-openshift-domain&#8250;:6443</strong>.
```

Comment:
```suggestion
          <strong>https://api.my-openshift.com:6443</strong>.
```
it's an example, it can be `https://example.com` no need to give a general form

Reply:
see: https://github.com/kubev2v/forklift-console-plugin/pull/865/files#diff-12c26a7b009e91f6d00cae0510ebb82188129d25b4ff19a284098486fe30e0fdR35